nomad/acl.go

```diff
@@ -10,9 +10,9 @@ import (
 	"github.com/hashicorp/nomad/nomad/structs"
 )
 
-// resolveToken is used to translate an ACL Token Secret ID into
+// ResolveToken is used to translate an ACL Token Secret ID into
 // an ACL object, nil if ACLs are disabled, or an error.
-func (s *Server) resolveToken(secretID string) (*acl.ACL, error) {
+func (s *Server) ResolveToken(secretID string) (*acl.ACL, error) {
```

Comment:
The rest of the changes are a purely mechanical renaming of resolveToken -> ResolveToken